scss/standalone/patterns_subnav.scss

```diff
@@ -8,6 +8,3 @@
 // used by navigation for accessibility links
 @import '../utilities_off-screen';
 @include vf-u-off-screen;
-
-@import '../patterns_subnav';
```

Comment:
Why removing this?

For now we still need to support old subnav pattern in Vanilla v2. So we should keep all the existing examples (just add "deprecated" to their titles, like we do with accordion for example) and add new examples with new class names.

Reply:
Fair play - my thinking was to remove the temptation for it to be used in a new instance between now and 3.0, and point people towards the new implementation, but I see your point.